src/components/PopupWithForm.js

```diff
@@ -24,4 +37,14 @@ export default class PopupWithForm extends Popup {
       this._popupForm.reset();
     });
   }
+
+  setHandleDelete(handleDelete, cardId) {
```

Comment:
Please, delete `setHandleDelete` from **universal** PopupWithForm. You should create `PopupWithConfirmation`, according to the checklist